packages/runtime-common/realm-permission-queries.ts

```diff
@@ -0,0 +1,61 @@
+import { DBAdapter } from './db';
+import { RealmPermissions } from './realm';
+
+async function insertPermission(
+  dbAdapter: DBAdapter,
+  realmURL: URL,
+  username: string,
+  read: boolean,
+  write: boolean,
+) {
+  let query = `INSERT INTO realm_user_permissions (realm_url, username, read, write) VALUES ($1, $2, $3, $4);`;
```

Comment:
I'd suggest using our `expressionToSql()` function in runtime-common/expression.ts. Specifically it has useful tools for dealing with params. You could express this like:
```
let query = ['INSERT INTO realm_user_permissions (realm_url, username, read, write) VALUES (', ...separatedByCommas([
  param(realmURL.href), param(username), param(read), param(write)
]), ')']; // you could even wrap `addExplicitParens()` too
```